test/integration/projects_test.go

```diff
@@ -0,0 +1,112 @@
+// Copyright 2025 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//go:build integration
+
+package integration
+
+import (
+	"context"
+	"os"
+	"testing"
+
+	"github.com/google/go-github/v75/github"
+)
+
+// Integration tests for Projects V2 endpoints defined in github/projects.go.
+//
+// These tests are intentionally defensive. They only require minimal
+// environment variables identifying a target org and user. Project numbers are
+// discovered dynamically by first listing projects and selecting one. For item
+// CRUD operations, the test creates a temporary repository & issue (where
+// possible) and adds/removes that issue as a project item. If prerequisites
+// (auth, env vars, permissions, presence of at least one project) are missing,
+// the relevant sub-test is skipped so other integration tests can still run.
+//
+// Required / optional environment variables:
+//   GITHUB_AUTH_TOKEN                  (required for any of these tests to run)
+//   GITHUB_TEST_ORG                    (org login; required for org project tests)
+//   GITHUB_TEST_USER                   (user login; required for user project tests)
+//   GITHUB_TEST_REPO                   (repo name)
```

Comment:
I propose splitting the comment into separate comments for `TestProjectsV2_Org` and `TestProjectsV2_User`. We don't need to mention `GITHUB_AUTH_TOKEN` since it's documented in the README: https://github.com/google/go-github/blob/b7f068fbeea594aba418a7e3699241dff979b45d/test/README.md?plain=1#L32.